### DISCUSSION TITLE
Contributions

### DISCUSSION THREAD

**Discussion by imskr** [mastodon/mastodon#23243] [General]
Hello, I wanted to contribute to mastodon open issues.

Is there any active or priority issues?

I have commented here: https://github.com/mastodon/mastodon/issues/20074 and waiting for confirmation

Open PR: https://github.com/mastodon/mastodon/pull/23055

**Comment by melroyvandenberg** [comment]
There is no structure or milestones. Nothing.. It looks like that Mastodon is lacking people in terms of planning overall and community support.
